Assignment1/retirement.py

```diff
@@ -1,10 +1,38 @@
-class Retirement:
-    def __init__(self):
-        pass
+def year_can_retire(age, salary, saving, goal):
+    """
+    Input user's current age, annual salary, percentage saved (savings matched
+    by employer - so double amount saved). Input desired retirement savings
+    goal. Output what age savings goal will be met. You can assume death at
+    100 years(therefore, indicate if the savings goal is not met).
+    :param age: current age of individual
+    :param salary:
+    :param saving:
+    :param goal:
+    :return:
+    """
 
-    def hello_world(self):
-        return "hello world"
+    if isinstance(age, (int, float)) is False:
+        raise ValueError("Age must be a number!")
 
+    if isinstance(salary, (int, float)) is False:
+        raise ValueError("Salary must be a number!")
 
-if __name__ == '__main__':
-    R = Retirement()
+    if isinstance(saving, (int, float)) is False:
+        raise ValueError("Saving must be a number!")
+
+    if isinstance(goal, (int, float)) is False:
+        raise ValueError("Goal must be a number!")
+
+    if age < 0:
+        raise ValueError("Can not have a negative age")
+
+    if salary <= 0:
+        raise ValueError("Can not have a positive salary")
```

Comment:
I think this should say "Can not have a negative salary" or "Salary must be greater than 0"